doc/user/upgrade/upgrade_guide.md

```diff
@@ -22,6 +22,30 @@ $ kubectl edit deployment/etcd-operator
 ### Incompatible upgrade
 In the case of an incompatible upgrade, the process requires restoring a new cluster from backup. See the [incompatible upgrade guide](incompatible_upgrade.md) for more information.
 
+## v0.6.0 -> v0.6.1
+
+**Note:** If your cluster does not have the following backup policy then you can simply upgrade the operator to the `v0.6.1` image.
+
+Backup policy:
+- A PV storage type: `backup.StorageType=PersistentVolume`
+- No storage class specified: `backup.pv.StorageClass` is unset
```

Comment:
Backup policy has `StorageType=PersistentVolume` but `pv.StorageClass` is unset:
```
... an example ..
```